content/docs/pipelines/installation/standalone-deployment.md

```diff
@@ -101,13 +108,37 @@ See the Google Kubernetes Engine (GKE) guide to
 
 1. Check the [Kubeflow Pipelines GitHub repository](https://github.com/kubeflow/pipelines/releases) for available releases.
 
-1. Upgrade to a version of Kubeflow Pipelines standalone:
+1. Upgrade to a version of Kubeflow Pipelines standalone is the same as deploying that version:
 
      ```
      export PIPELINE_VERSION=<version-you-want-to-upgrade-to>
+     kubectl apply -k github.com/kubeflow/pipelines/manifests/kustomize/cluster-scoped-resources?ref=$PIPELINE_VERSION
+     kubectl wait --for condition=established --timeout=60s crd/applications.app.k8s.io
      kubectl apply -k github.com/kubeflow/pipelines/manifests/kustomize/env/dev?ref=$PIPELINE_VERSION
      ```
 
+     This is for 0.4.0 and above, commands for older versions can be found at [Deploying Kubeflow Pipelines](#deploying-kubeflow-pipelines)
+
+1. Delete removed resources manually
+
+     This depends on which version you are upgrading from and to.
```

Comment:
Since we only have one set of instructions, consider removing this sentence.

Reply:
The implied other possibility is, users don't need to do anything specific if they upgrade to other versions..

Reply:
Here is a suggested revision:

Depending on the version you are upgrading from and the version you are upgrading to, some Kubeflow Pipelines resources may have become obsolete.